### ISSUE TITLE
bug error

### ISSUE THREAD

**Issue by cuihaijun** [nextcloud/android#9871]
### Steps to reproduce
1. 
2. 
3. 

### Expected behaviour
- Tell us what should happen

### Actual behaviour
- Tell us what happens

### Can you reproduce this problem on https://try.nextcloud.com?
- Please create a test demo account and see if this still happens there.
- If yes, please open up a bug report
- If not, please verify server setup and ask for help on forum

### Environment data
Android version:

Device model: 

Stock or customized system:

Nextcloud app version:

Nextcloud server version:

Reverse proxy:

#### Web server error log
```
Insert your webserver log here
```

#### Nextcloud log (data/nextcloud.log)
```
Insert your Nextcloud log here
```
**NOTE:** Be super sure to remove sensitive data like passwords, note that everybody can look here! You can use the Issue Template application to prefill some of the required information: https://apps.nextcloud.com/apps/issuetemplate

**Comment by cuihaijun** [comment]
************ CAUSE OF ERROR ************

java.lang.RuntimeException: Unable to pause activity {com.nextcloud.client/com.owncloud.android.ui.activity.FileDisplayActivity}: java.lang.IllegalArgumentException: Receiver not registered: de.cotech.hw.internal.transport.usb.UsbConnectionDispatcher$1@19b13f5
	at android.app.ActivityThread.performPauseActivityIfNeeded(ActivityThread.java:5425)
	at android.app.ActivityThread.performPauseActivity(ActivityThread.java:5374)
	at android.app.ActivityThread.handlePauseActivity(ActivityThread.java:5314)
	at android.app.servertransaction.TransactionExecutor.performLifecycleSequence(TransactionExecutor.java:242)
	at android.app.servertransaction.TransactionExecutor.cycleToPath(TransactionExecutor.java:215)
	at android.app.servertransaction.TransactionExecutor.executeLifecycleState(TransactionExecutor.java:187)
	at android.app.servertransaction.TransactionExecutor.execute(TransactionExecutor.java:105)
	at android.app.ActivityThread$H.handleMessage(ActivityThread.java:2613)
	at android.os.Handler.dispatchMessage(Handler.java:110)
	at android.os.Looper.loop(Looper.java:219)
	at android.app.ActivityThread.main(ActivityThread.java:8668)
	at java.lang.reflect.Method.invoke(Native Method)
	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:513)
	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1109)
Caused by: java.lang.IllegalArgumentException: Receiver not registered: de.cotech.hw.internal.transport.usb.UsbConnectionDispatcher$1@19b13f5
	at android.app.LoadedApk.forgetReceiverDispatcher(LoadedApk.java:1662)
	at android.app.ContextImpl.unregisterReceiver(ContextImpl.java:1671)
	at android.content.ContextWrapper.unregisterReceiver(ContextWrapper.java:664)
	at de.cotech.hw.internal.transport.usb.UsbConnectionDispatcher.onInactive(UsbConnectionDispatcher.java:131)
	at de.cotech.hw.SecurityKeyManager$DispatcherActivityLifecycleCallbacks.ensureStateInactive(SecurityKeyManager.java:316)
	at de.cotech.hw.SecurityKeyManager$DispatcherActivityLifecycleCallbacks.refreshActiveState(SecurityKeyManager.java:290)
	at de.cotech.hw.SecurityKeyManager$DispatcherActivityLifecycleCallbacks.onActivityPaused(SecurityKeyManager.java:333)
	at android.app.Application.dispatchActivityPaused(Application.java:473)
	at android.app.Activity.dispatchActivityPaused(Activity.java:1360)
	at android.app.Activity.onPause(Activity.java:2333)
	at androidx.fragment.app.FragmentActivity.onPause(FragmentActivity.java:388)
	at com.owncloud.android.ui.activity.BaseActivity.onPause(BaseActivity.java:84)
	at com.owncloud.android.ui.activity.FileActivity.onPause(FileActivity.java:253)
	at com.owncloud.android.ui.activity.FileDisplayActivity.onPause(FileDisplayActivity.java:1254)
	at android.app.Activity.performPause(Activity.java:8400)
	at android.app.Instrumentation.callActivityOnPause(Instrumentation.java:1529)
	at android.app.ActivityThread.performPauseActivityIfNeeded(ActivityThread.java:5415)
	... 13 more

************ APP INFORMATION ************
ID: com.nextcloud.client
Version: 30170090
Build flavor: gplay

************ DEVICE INFORMATION ************
Brand: HUAWEI
Device: HWNOH
Model: NOH-AN00
Id: HUAWEINOH-AN00
Product: NOH-AN00

************ FIRMWARE ************
SDK: 29
Release: 10
Incremental: 102.0.0.220C00

**Comment by AlvaroBrey** [comment]
This has been fixed since 3.17.1. Please update your app.

**Comment by AlvaroBrey** [comment]
Duplicate of #8585
